lib/kinetic.js

```diff
@@ -488,6 +490,9 @@ class PDU extends stream.Readable {
         if (this._protobuf.authType === 1 && !this.checkHmacIntegrity(
                     this.getSlice(this._protobuf.hmacAuth.hmac)))
             throw propError("hmacFail", "HMAC does not match");
+        if (this.getMessageType() === null) {
```

Comment:
No, normally when the protobuf is decoded; all the unset values are set to null.
And this PDU is the initial pdu that have to be `null` for messageType.